### DISCUSSION TITLE
Starting with free weapons at start/end of levels, how do I turn this off?

### DISCUSSION THREAD

**Discussion by qwertyuiopmememe** [Novum/vkQuake#748] [Q&A] [answered]
I want to play the Quake 1 normally without this assistance, is there any way to disable this or is it part of vkQuake?

**Comment by qwertyuiopmememe** [comment]
I have no idea how to fix this

**Comment by vsonnier** [comment, marked as answer]
> I have no idea how to fix this

There is nothing to fix, as it is the logic (probably in QuakeC) implemented by the mod / game itself while playing continuous maps in episodes. I'm not going to try to change / hack this, sorry.
